packages/rest/src/rest.application.ts

```diff
@@ -83,6 +84,16 @@ export class RestApplication extends Application implements HttpServerLike {
     this.restServer.handler(handlerFn);
   }
 
+  /**
+   * Mount static assets to the REST server
+   * @param path The path to serve the asset
+   * @param root The root directory
+   * @param options Options for serve-static
+   */
+  asset(path: string, root: string, options?: ServeStaticOptions) {
```

Comment:
Can we rename this to `static` -- it's traditional and what developers would expect. Asset to me can imply anything ... a controller can be an asset imo.